applications/CompressiblePotentialFlowApplication/tests/cpp_tests/test_incompressible_potential_element.cpp

```diff
@@ -73,9 +95,14 @@ namespace Kratos {
 
       // Check the RHS values (the RHS is computed as the LHS x previous_solution,
       // hence, it is assumed that if the RHS is correct, the LHS is correct as well)
-      KRATOS_CHECK_NEAR(RHS(0), 0.5, 1e-7);
-      KRATOS_CHECK_NEAR(RHS(1), 0.0, 1e-7);
-      KRATOS_CHECK_NEAR(RHS(2), -0.5, 1e-7);
+      std::vector<double> reference(3);
```

Comment:
```suggestion
      std::vector<double> reference({0.5, 0.0, -0.5});
```
This should work and you avoid the three lines below.